### PR TITLE
MINOR: Remove outdated comment in Connect's WorkerCoordinator

### PR DESCRIPTION
**Pull request by C0urante**
Connect supports multiple assignment strategies now.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java [hunk 1]
```diff
@@ -50,7 +50,6 @@
  * to workers.
  */
 public class WorkerCoordinator extends AbstractCoordinator implements Closeable {
-    // Currently doesn't support multiple task assignment strategies, so we just fill in a default value
     public static final String DEFAULT_SUBPROTOCOL = "default";
```
Comment (chia7712) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java]: It seems ```DEFAULT_SUBPROTOCOL``` is used by testing only. Should we move it to test code?

**Review by chia7712 — APPROVED** [review]
@C0urante Thanks for your patch. LGTM

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java (hunk 1 above)
Comment (kkonstantine) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java]: The definition has moved into: 
https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/ConnectProtocolCompatibility.java#L45

We should use `EAGER.protocol()` to replace the usage of this member field variable and then remove it.

**Review by kkonstantine — APPROVED** [review]
LGTM with a note on how to replace the member variable in tests.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java (hunk 1 above)
Comment (C0urante) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java]: Sounds good, done.

File: connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinatorTest.java [hunk 2]
```diff
@@ -105,7 +106,7 @@
     @Parameters
     public static Iterable<?> mode() {
         return Arrays.asList(new Object[][]{
-                {ConnectProtocolCompatibility.EAGER, 1},
+                {EAGER, 1},
                 {ConnectProtocolCompatibility.COMPATIBLE, 2}});
```
Comment (kkonstantine) [inline connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinatorTest.java]: If you change the line above, then let's change this one too and fix the asymmetry

**Pull request merged**